service_mesh/service_mesh_user_guide/ossm-observability.adoc

```diff
@@ -0,0 +1,25 @@
+[id="ossm-observability"]
+= Data visualization and observability 
+include::modules/ossm-document-attributes.adoc[]
+:context: observability
+
+toc::[]
+
+You can view your application's topology, health and metrics in the Kiali console. If your service is having issues, the Kiali console offers ways to visualize the data flow through your service. You can view insights about the mesh components at different levels, including abstract applications, services, and workloads. It also provides an interactive graph view of your namespace in real time. 
+
+.Prerequisites
+
+* You can observe the data flow through an example application if you install the xref:../../service_mesh/service_mesh_day_two/ossm-example-bookinfo.html#ossm-tutorial-bookinfo-overview_ossm-bookinfo-tutorial[Bookinfo sample application]. 
```

Comment:
Possible edit here for the Prerequisites (because they can view their own application data, so Bookinfo isn't technically a prerequisite).
* Kiali Operator installed and the Kiali application deployed.
* At least one application added to the mesh.  You can use the Bookinfo sample application.

Reply:
I removed the prerequisite heading and turned it into paragraph form.